frontends/web/src/hooks/app.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2021 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useContext } from 'react';
+import { ConfigContext, ConfigContextInterface } from '../contexts/ConfigContext';
+
+export const useConfig = (): ConfigContextInterface => {
+    return useContext(ConfigContext);
+}
```

Comment:
Why not `hooks/config.ts`?
Or what is the plan with `hooks/app.ts` is the idea that more goes into app hook?

Reply:
Yes, its better to have a separate file, i renamed it :+1: